app/services/irs_attempts_api/tracker.rb

```diff
@@ -1,11 +1,10 @@
 module IrsAttemptsApi
   class Tracker
-    attr_reader :session_id, :enabled_for_session, :request, :user, :sp, :cookie_device_uuid,
+    attr_reader :enabled_for_session, :request, :user, :sp, :cookie_device_uuid,
                 :sp_request_uri, :analytics
 
-    def initialize(session_id:, request:, user:, sp:, cookie_device_uuid:,
+    def initialize(request:, user:, sp:, cookie_device_uuid:,
                    sp_request_uri:, enabled_for_session:, analytics:)
-      @session_id = session_id # IRS session ID
       @request = request
```

Comment:
I think when we say this can be an empty class, I mean totally empty.

Like all we need is:

```ruby
module IrsAttemptsApi
  class Tracker
    include TrackerEvents

    def track_event(_event_type, _metadata = {})
    end
  end 
end
```